pkg/controller/instanceset/reconciler_instance_alignment.go

```diff
@@ -159,10 +174,10 @@ func (r *instanceAlignmentReconciler) Reconcile(tree *kubebuilderx.ObjectTree) (
 		if _, ok := deleteNameSet[pod.Name]; !ok {
 			continue
 		}
```

Comment:
From a true "serial update" perspective, insert and delete operations cannot occur simultaneously. 

If isOrderedReady is true, concurrency should be 1, but here it is 2. 

We can consider treating isOrderedReady as true as equivalent to concurrency being 1. This way, the two logics can be combined.

@free6om

Reply:
`concurrency` defaults to 1 sounds good to me.

In fact, there is no concurrency thing in `OrderedReady` policy: all actions are handled in-order. the `concurrency` variable should be removed if only having `OrderedReady` policy.